src/lib/utilities/route-for-api.ts

```diff
@@ -57,13 +57,19 @@ const base = (namespace?: string): string => {
   return baseUrl;
 };
 
-const withBase = (endpoint: string, namespace?: string): string => {
+const getPath = (endpoint: string): string => {
   if (endpoint.startsWith('/')) endpoint = endpoint.slice(1);
+  return `/api/v1/${endpoint}`;
+};
+
+const withBase = (path: string, namespace?: string): string => {
   const baseUrl = base(namespace);
```

Comment:
I don't see `base` in _this_ PR, but it feels like it could just do all of this logic?

Reply:
Kept it separate so that I could use `base` in `fetchWorkflowForRunId` [here](https://github.com/temporalio/ui/pull/1404/files#diff-bec5e9fc4492624aa6906c9c6b0567006dbe866be6d0eda43af03204166e0101R179).